### PR TITLE
Podseeker nerf (against non-pods)

### PR DESCRIPTION
**Pull request by Tarmunora**
## About the PR 
Changes podseekers to only explode if they are seeking a target. otherwise, they just do (hefty) blunt damage and disorient

## Why's this needed? 
Podseekers see more use as a 'fuck this guy in particular' ammo rather than as anti-pod ammo, which is rather silly.

## Changelog

```
(u)Tarmunora:
(*)Changed pod(etc)seeking 40mmHE shells to have a high chance of not detonating on impact unless they have found a target to seek. Additionally boosted their direct-impact stats to compensate somewhat.
```

### CODE REVIEW

File: code/modules/projectiles/bullet.dm [hunk 1]
```diff
@@ -674,9 +674,22 @@ toxic - poisons
 
 	seeker
 		name = "drone-seeking grenade"
+		power = 50 //even if they don't explode, you FEEL this one
 		var/max_turn_rate = 20
 		var/type_to_seek = /obj/critter/gunbot/drone //what are we going to seek
 		precalculated = 0
+		on_hit(atom/hit, angle, var/obj/projectile/P)
+			if (P.data || prob(10)) //maybe
```
Comment (ZeWaka) [inline code/modules/projectiles/bullet.dm:682]: Wouldn't this make it not hit pods properly?

Comment (Tarmunora) [reply code/modules/projectiles/bullet.dm:682]: P.data is non-null if it is seeking a pod, so it'll go up to the parent proc in `/datum/projectile/bullet/autocannon` where the explosion is done.

**Comment by Kyle2143** [comment]
I don't think that this is required. This ammunition is the only type of explosive ammunition available to security and sometimes that is needed, specifically when dealing with nuke ops.

I'd be more amenable to this change if there was another type of ammunition that security or the HoS had access to that was explosive. Maybe slightly less explosive would do and without the tracking. I'd meant to add a non-tracking explosive ammunition to the game a few months ago, but stuff got in the way.

**Comment by mybluecorners** [comment]
i am in support of this change, i *do* think it is needed because the current way the pod seeker works is extremely not-fun for whatever human is on the receiving end (you just instantly gib, usually)
An alternate type of explosive ammunition for security would be nice but i think it is outside the scope of this patch.

**Comment by Flaborized** [comment]
I think this change is good, podseekers in practice are extremely silly. If sec wants an explosive weapon, they can more easily grab stingers and stuff them in a riot launcher (which is way more fun to use and easier for sec to grab) so I don't really see this as "removing an important part of the sec kit"

**Comment by flrsh** [comment]
I agree with mbc and Flab. Right now, podseekers are rarely used for their intended purpose and aren't very fun.

**Comment by Enakai** [comment]
Mmmh, I'm a lil' bit against the podseeker change against non-pods, just because podseekers are really useful for dealing with super impossible targets. They're only really pulled out, at least by myself, when it's something that I cannot absolutely handle. They aren't even all that reliable - if you're anywhere near a pod, it'll usually fling off in the direction of that pod, so it's absolutely a last resort. The only situations that I see them really cropping up and destroying players were on ass day and that was it.

**Comment by Tarmunora** [comment]
Would need to be added to maps, but these (40mm doorknocker) rounds could fill that role without being an outright gib, which doesn't quite feel on-brand for security to me

**Comment by UrsulaMejor** [comment]
Podseekers aren't used for their intended purpose because tasers are better than them at killing pods. Why waste explosive ammo when emptying a taser or two destroys most vehicles?

This change is bad, imo. Sec needs some kind of explosive weaponry, and I don't actually see them get abused all that often, except by ass day HoS with EOD suit (which isn't a thing anymore)

Make them still explode but not gib. give them a damage bonus to pods to make then worth using.

**Comment by Tarmunora** [comment]
> Sec needs some kind of explosive weaponry ... Make them still explode, but not gib
Security has stinger grenades which can be loaded into the riot launchers to do exactly this as-is

**Comment by ryebek** [comment]
I'd rather things stayed as is, as I can honestly say, this is the only thing I use them for, because they're trash against pods, compared to a power hammer. They're super deadly, yes. But you also only get 4 shots, and you're lucky if it doesn't fuck off somewhere as soon as it detects a pod. They're one of the best weapons against nukies, and I only pull them out when its absolutely necessary as HoS, due to their scarcity. I think the armory should be well equipped, and in my eyes, this is security's RPG. Although maybe blacklist the ammo from the NTSO ammo pouch, because I abused that a bunch, since it lets you hold both of the normal sized ammo in your pocket.

**Comment by ZeWaka** [comment]
This was never conceived to be "Security's RPG." If you want a tool like that, add a method to obtain the doorknocker rounds.

**Pull request merged**